middleware/tracing/span_test.go

```diff
@@ -66,43 +66,75 @@ func Test_parseFullMethod(t *testing.T) {
 
 func Test_peerAttr(t *testing.T) {
 	tests := []struct {
-		name string
-		addr string
-		want []attribute.KeyValue
+		name     string
+		addr     string
+		isClient bool
+		want     []attribute.KeyValue
 	}{
 		{
-			name: "nil addr",
-			addr: ":8080",
+			name:     "nil addr with client context",
+			addr:     ":8080",
+			isClient: true,
 			want: []attribute.KeyValue{
-				semconv.NetPeerIPKey.String("127.0.0.1"),
-				semconv.NetPeerPortKey.String("8080"),
+				semconv.NetworkPeerAddressKey.String("127.0.0.1"),
+				semconv.ServerPortKey.Int(8080),
```

Comment:
Will this result in inconsistent final results, or is it just a change in method name